### DISCUSSION TITLE
Is it possible to add hook per build?

### DISCUSSION THREAD

**Discussion by tkxkd0159** [goreleaser/goreleaser#3612] [Q&A] [answered]
I am struggling cross-compiling using cgo with goreleaser. I have to link static library for my binary depend on its GOARCH opts.  
For example, I have `libmylib.aarch64.a` and `libmylib.x86_64.a` and I have to rename them to `libmylib.a` when I link this library at build time. but In goreleaser, it seems to be no way to make this change on the build stage. If there are some method to resolve this issue, please help me. Thanks!

**Comment by caarlos0** [comment, marked as answer]
have you check https://goreleaser.com/cookbooks/cgo-and-crosscompiling/ ?

**Reply by tkxkd0159** [reply]
Thank you for your comment and yes I already checked it and I may be able to resolve my issue if I use goreleaser-pro. so I just wonder if I can handle this issue with goreleaser normal version. Is there any method to handle this?  
The painful point is that goreleaser build & release at once.
